engine/swarm/store_sensitive_strings.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Manage sensitive strings (secrets) for Docker services
+description: How to securely store, retrieve, and use sensitive strings (secrets) in Docker services
```

Comment:
String has a very specific meaning for developers and programmers. I think we want to avoid using that term. The term `blob` would technically be more adequate, but I think `sensitive data` reads a lot better.